Sources/FluentUI_common/Core/Theme/Tokens/DynamicColor.swift

```diff
@@ -6,30 +6,31 @@
 import SwiftUI
 
 /// A container that stores a dynamic set of `Color` values.
-struct DynamicColor: Hashable {
+@objc(MSFDynamicColor)
+public final class DynamicColor: NSObject {
 
     /// Creates a custom `ShapeStyle` that stores a dynamic set of `Color` values.
     ///
     /// - Parameter light: The default `Color` for a light context. Required.
     /// - Parameter dark: The override `Color` for a dark context. Optional.
     /// - Parameter darkElevated: The override `Color` for a dark elevated context. Optional.
-    init(light: Color,
-         dark: Color? = nil,
-         darkElevated: Color? = nil) {
-        self.light = light
+	public init(light: Color,
+				dark: Color? = nil,
+				darkElevated: Color? = nil) {
```

Comment:
Alignment looks off? Or is that just Github?

Reply:
Good eye, it's because the file was a mix of tabs and spaces, and github uses 8-space tab characters by default for some reason. Converting to all spaces.